src/modules/0.0.5/aws_ec2/index.ts

```diff
@@ -28,11 +30,12 @@ export const AwsEc2Module: Module = new Module({
   mappers: {
     instance: new Mapper<Instance>({
       entity: Instance,
-      equals: (a: Instance, b: Instance) => Object.is(a.name, b.name) &&
-        Object.is(a.instanceId, b.instanceId) &&
+      equals: (a: Instance, b: Instance) => Object.is(a.instanceId, b.instanceId) &&
         Object.is(a.ami, b.ami) &&
         Object.is(a.instanceType, b.instanceType) &&
         Object.is(a.keyPairName, b.keyPairName) &&
+        Object.is(Object.keys(a.tags)?.length, Object.keys(b.tags)?.length) &&
+        Object.keys(a.tags)?.every(ak => a.tags[ak] === b.tags[ak]) &&
```

Comment:
Since tags could be null I thing this should be:
```suggestion
        Object.keys(a.tags ?? {}).every(ak => a.tags[ak] === b.tags[ak]) &&
```